Project.toml

```diff
@@ -18,6 +19,7 @@ XMLDict = "228000da-037f-5747-90a9-8195ccbf91a5"
 
 [compat]
 AWS = "1.25"
+Compat = "3.29.0"
```

Comment:
minimum version to get https://github.com/JuliaLang/Compat.jl/pull/742 (but this syntax means this version or any later in the v3 series)